metrics/tags_test.go

```diff
@@ -90,6 +91,25 @@ func TestTagSets(t *testing.T) {
 	assert.True(t, tags4.Contains(tags4))
 }
 
+func TestBigTagSetMarshalJSON(t *testing.T) {
+	t.Parallel()
+
+	expextedJSON := `{"tag0":"value0","tag1":"value1","tag2":"value2","tag3":"value3","tag4":"value4","tag5":"value5","tag6":"value6","tag7":"value7"}`
+	assert.Greater(t, len(expextedJSON), 128)
+
+	r := NewRegistry()
+	root := r.RootTagSet()
+
+	tags := root
```

Comment:
```suggestion
	tags := NewRegistry().RootTagSet()
```
No need for the extra variable if we won't use it. :)